zerver/lib/notifications.py

```diff
@@ -274,7 +274,8 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
         })
 
     from zerver.lib.email_mirror import create_missed_message_address
-    address = create_missed_message_address(user_profile, missed_messages[0])
+    reply_to_address = create_missed_message_address(user_profile, missed_messages[0])
+    reply_to_name = None if reply_to_address == FromAddress.NOREPLY else "Zulip"  # type: Optional[Text]
```

Comment:
This code is hard to read.  As a policy matter, we actually almost never use the inline `if` feature of Python for this reason.